tezos/context/src/working_tree/storage.rs

```diff
@@ -399,175 +444,478 @@ impl TryFrom<usize> for InodeId {
     }
 }
 
+impl From<u32> for InodeId {
+    fn from(v: u32) -> Self {
+        Self(v)
+    }
+}
+
+impl From<InodeId> for u32 {
+    fn from(v: InodeId) -> Self {
+        v.0
+    }
+}
+
+#[derive(Clone, Debug, Copy, PartialEq, Eq)]
+pub struct FatPointerId(u32);
+
+impl TryInto<usize> for FatPointerId {
```

Comment:
@sebastiencs when can this fail?

Reply:
Ah yes, it cannot fail. I will change the trait to `Into`